pagopa/io-app
# chore: [IOCOM-1684] New FIMS IAB error and error handling structure

## Short description
Added said UI, also reworked the way FIMS SSO handles errors to be more centralized and expandable on.
<img width="250" alt="Screenshot 2024-09-26 at 16 30 44" src="https://github.com/user-attachments/assets/7c66b585-d4c0-41d5-b664-9723cbf692b7">

## List of changes proposed in this pull request
- addition of required I18n
- updated the reducer to include an error tag in case of error
- updated all concerned actions
- moved error logic in a separate file

## How to test
modify `handleFimsGetConsentsList.ts` to include this snippet around line 31: 
```ts
  yield* put(
    fimsGetConsentsListAction.failure({
      errorTag: "MISSING_INAPP_BROWSER",
      standardMessage: "",
      debugMessage: ""
    })
  );
  return;
```
then navigate to the FIMS SSO flow, and make sure the screenshotted UI is shown.
